### ISSUE TITLE
Test dataset for CATHS40

### ISSUE THREAD

**Issue by haniulu** [flatironinstitute/deepblast#169]
Hi, the CATHS40 training database is provided in the article, and I would like to know if it is possible to provide the protein of the test set and validation set of the CATHS40 mentioned in the paper experiment.
